docker-compose.yml

```diff
@@ -136,41 +174,43 @@ services:
       api:
         condition: service_healthy
     command: ['./scripts/start.worker.sh']
+    environment:
+      <<: *common-environment
+      LAGO_REDIS_CACHE_URL: "redis://${LAGO_REDIS_CACHE_HOST}:${LAGO_REDIS_CACHE_PORT}"
+      DATABASE_URL: "postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}?search_path=${POSTGRES_SCHEMA}"
+      REDIS_URL: "redis://${REDIS_HOST}:${REDIS_PORT:-6379}"
     healthcheck:
       test: ['CMD-SHELL', 'bundle exec sidekiqmon | grep $(hostname) || exit 1']
+    volumes:
+      - lago_storage_data:/app/storage
+    
+
+  api-clock:
+    container_name: lago-clock
+    image: getlago/api:v1.13.1
```

Comment:
is it possible to use a global en var for the version number?

Reply:
sure ! Fix in last commit :)